js/ui/iconGridLayout.js

```diff
@@ -148,6 +158,7 @@ const IconGridLayout = new Lang.Class({
     _getDefaultIcons: function() {
         let mergedJson = this._mergeJsonStrings(
             this._loadConfigJsonString(DEFAULT_CONFIGS_DIR, DEFAULT_CONFIG_NAME_BASE),
+            this._loadConfigJsonString(OVERRIDE_CONFIGS_DIR, OVERRIDE_CONFIG_NAME_BASE),
```

Comment:
I am not sure this will always do what you want; `mergeJsonStrings()` will create an "union" of all the settings specified in the various files, so if e.g. the `/usr` config for a language specifies a folder that is not in the `/var` config, the end result is that such folder will be in the merged JSON.
Instead, the code should load either the JSON from `/usr` or the one in `/var`, excluding the one in `/usr` in the latter case, then apply the append/prepend overrides to the result.